stablehorde_api/models.py

```diff
@@ -1,22 +1,79 @@
 from typing import Optional, Sequence
+import logging
 
 import msgspec
 
+class ModelPayloadLorasStable(msgspec.Struct):
+    name: str # The exact name or CivitAI ID of the LoRa.
+    model: int | None = None # The strength of the LoRa to apply to the SD model.
+    clip: int | None = None # The strength of the LoRa to apply to the clip model.
```

Comment:
Formatting - 2 spaces before inline comment
```suggestion
    name: str  # The exact name or CivitAI ID of the LoRa.
    model: int | None = None  # The strength of the LoRa to apply to the SD model.
    clip: int | None = None  # The strength of the LoRa to apply to the clip model.
```